src/Cli - Ex 3/cliTodoFunctions.js

```diff
@@ -0,0 +1,158 @@
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import fs from 'fs';
+import PokemonClient from './PokemonClient.js';
+import imageToAscii from 'image-to-ascii';
```

Comment:
its not in your package json file
![image](https://user-images.githubusercontent.com/65164924/173235170-ee4c17b6-bebf-4fb1-beae-c2c88101dda3.png)

Reply:
you should of seperated your code into different responsibilities and you could even use some of them from the last 2 steps
1. cli - handles input and output to the console
2. TodoListManager - connects all the 
3. Items manager - handle all operations on the list, calls pokemon client and filesManager
4. Files manager - handle all read and writes from the file and returns a list
all of those are crucial for good separation and easy refactoring later on
lets say you want to change the client to web app instead of the console - I would expect the change only to replace the cliService file and all others should be the same
same goes for changing the file to a database